docs/dev/driver.rst

```diff
@@ -162,6 +162,27 @@ The driver object contains the following attributes:
       String containing the distribution ID of Firefox. This value is
       ``undefined`` on Firefox versions older than 48.0.
 
+.. js:function:: addons.get(addonId)
```

Comment:
They should be here, yes. They should have been added in #778, but that PR got merged without docs for some reason.